README.md

```diff
@@ -111,6 +111,10 @@ The `from` configuration property causes Binci to use the specified image to run
 
 For testing different images easily, the either the `-b <build-dockerfile>` or `-f <from-alternate-image>` arguments can be passed on execution.
 
+## Tags (`tags <Array<string>>`)

```

Comment:
can you add a snippet/example of this and explanation of results?

Reply:
Good call!

Reply:
Added :boom: